weld/subsystem/src/main/java/org/jboss/as/weld/WeldResourceDefinition.java

```diff
@@ -67,6 +70,14 @@
             .setRestartAllServices()
             .build();
 
+    static final SimpleAttributeDefinition THREAD_POOL_SIZE_ATTRIBUTE =
+            new SimpleAttributeDefinitionBuilder(THREAD_POOL_SIZE, ModelType.INT, true)
+            .setAllowExpression(true)
+            .setValidator(new IntRangeValidator(1))
+            .setDefaultValue(new ModelNode(WeldExecutorServices.DEFAULT_BOUND))
```

Comment:
This isn't really true. This isn't a true constant default, it's just the value that would be used on the process the user is asking. Which in the case of a managed domain is likely the DC, which is on a machine that may not have any servers at all running, and very likely isn't the only machine with servers.

I don't think it's necessary to report a default value for this.